m2cgen/interpreters/java/code_generator.py

```diff
@@ -31,13 +33,16 @@ def class_definition(self, model_name):
         self.add_block_termination()
 
     @contextlib.contextmanager
-    def method_definition(self, name, args, return_type, modifier="public"):
+    def method_definition(self, name, args, return_vector, modifier="public"):
```

Comment:
`return_vector`? I'm not sure I understand what does that mean. We might return non-vector value as well.

Reply:
Ah, it's boolean. Let's rename it to something that would tell so.

Reply:
Done